src/python/module/z5py/converter.py

```diff
@@ -73,7 +73,7 @@ def convert_to_h5(in_path, out_path,
         # if fit_to_roi == True
         blocks = blocking(shape, block_shape, roi, fit_to_roi)
         if roi is not None:
-            roi = normalize_slices(roi, shape)
+            roi, to_squeeze = normalize_slices(roi, shape)
```

Comment:
Please change to `roi, _ = normalize_slices(...)` to avoid PEP8 unused variable.